resources/js/processes/modeler/components/inspector/SignalPayload.vue

```diff
@@ -106,6 +106,7 @@ export default {
   methods: {
     payloadChange(selectedObject) {
       this.config.payload[0].id = selectedObject.id;
+      this.$set(this.node().eventDefinitions[0], "config", JSON.stringify(this.config));
```

Comment:
Please do not use `this.$set`, because it will skip the InspectorHandler logic.
Change the logic to use `this.$emit`